projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,106 @@
+  "tooltip.enderite.enderite_pickaxe": "更快地挖掘末地石",
+  "tooltip.enderite.enderite_pickaxe_configurable": "你可以在 “data/forge/tags/blocks/end_stones.json”中配置可以让末影合金镐破坏更快的方块",
+  "tooltip.enderite.enderite_armor_set_bonus": "套装效果:",
+  "tooltip.enderite.enderite_armor_set_bonus_is_endermite_peaceful": "使末影螨不会主动攻击你",
+  "tooltip.enderite.enderite_armor_set_bonus_slow_falling": "按下 “Shift”,获得缓降效果",
+  "tooltip.enderite.enderite_armor_set_bonus_over_value": "如果护甲耐久低于",
+  "tooltip.enderite.enderite_helmet": "使你的注视不会激怒末影人",
+  "tooltip.enderite.enderite_elytra_chestplate": "你能飞了！",
+  "tooltip.enderite.enderite_leggings_press_shift": "按下 “Shift” ,解除已经获得的漂浮效果",
+  "tooltip.enderite.enderite_leggings_op_mode": "移除漂浮效果",
+  "tooltip.enderite.enderite_boots": "让你在末地石上移动得更快",
+  "tooltip.enderite.enderite_boots_op_mode": "允许你在末地更快地移动",
+  "tooltip.enderite.enderite_boots_configurable": "您可以在 “data/forge/tags/blocks/enderite_boots_walks_faster_on_blocks.json”中配置获得速度效果的方块",
+
+
+  "advancement.end.enderite.charge_end_respawn_anchor.title": "感觉像家一样！",
+  "advancement.end.enderite.charge_end_respawn_anchor.description": "将末影重生锚充能至最大",
+  "advancement.end.enderite.explore_all_end_biomes.title": "寒冷的旅途目的地",
+  "advancement.end.enderite.explore_all_end_biomes.description": "探索所有末地生物群系",
+  "advancement.end.enderite.obtain_endercrystal_shard.title": "末影水晶碎片？",
+  "advancement.end.enderite.obtain_endercrystal_shard.description": "找到一个末影水晶洞穴！",
+  "advancement.end.enderite.obtain_enderite_ingot.title": "这样好吗？",
+  "advancement.end.enderite.obtain_enderite_ingot.description": "获得末影合金锭",
+  "advancement.end.enderite.obtain_enderite_upgrade_smithing_template.title": "升级？",
+  "advancement.end.enderite.obtain_enderite_upgrade_smithing_template.description": "获得一个末影升级锻造模板",
+  "advancement.end.enderite.destroyer_of_the_end.title": "末地毁灭者！",
+  "advancement.end.enderite.destroyer_of_the_end.description": "获得末影合金镐",
+  "advancement.end.enderite.obtain_enderite_armor.title": "我现在是末影人了吗？",
+  "advancement.end.enderite.obtain_enderite_armor.description": "获得一套完整的末影合金盔甲",
+  "advancement.end.enderite.obtain_enderite_elytra_chestplate.title": "完美的盔甲",
+  "advancement.end.enderite.obtain_enderite_elytra_chestplate.description": "获得末影合金下界合金胸甲",
+
+  "advancement.husbandry.enderite.obtain_enderite_hoe.title": "好的，当你能做到的时候！？",
+  "advancement.husbandry.enderite.obtain_enderite_hoe.description": "获得末影合金锄",
+
+  "upgrade.enderite.enderite_upgrade": "末影合金升级",
+  "item.enderite.smithing_template.enderite_upgrade.additions_slot_description": "在这里放一个末影合金锭或一个鞘翅。",
+  "item.enderite.smithing_template.enderite_upgrade.applies_to": "末影合金装备，末影合金鞘翅胸甲",
+  "item.enderite.smithing_template.enderite_upgrade.base_slot_description": "在这里放置一件末影合金盔甲、武器或工具，或者把末影合金鞘翅胸甲放在这里。",
+  "item.enderite.smithing_template.enderite_upgrade.ingredients": "末影合金锭，鞘翅",
+  "trim_material.enderite.enderite": "末影合金材料（类似漂浮I）",
+
+
+
+  "item.enderite.gilded_enderite_helmet": "镶金的末影合金头盔",
```

```suggestion
  "item.enderite.gilded_enderite_helmet": "镶金末影合金头盔",
```

见原版`镶金黑石`
[原版译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)

原本译名问题请全文通查，所有内容都要查，确保没有不一致的情况